main.sh

```diff
@@ -24,6 +24,7 @@ function _do_it_for_all() {
         bash "npm.sh" "$what_to_do"
         bash "dropbox.sh" "$what_to_do"
         bash "git.sh" "$what_to_do"
+        bash "docker.sh" "$what_to_do"
```

Comment:
This should instead lie in the section below to be called with `sudo`.
Also, `sudo` shouldn't be used in `docker.sh` file, but that file invoked from main with `sudo` itself.